src/main/java/org/sagebionetworks/web/client/widget/entity/MarkdownWidgetViewImpl.java

```diff
@@ -41,28 +42,39 @@ public interface Binder extends UiBinder<Widget, MarkdownWidgetViewImpl> {}
   @UiField
   Italic emptyPanel;
 
+  private GWTWrapper gwt;
   public static GlobalApplicationState globalAppState;
   public static final EventListener relativeLinkClickHandler = event -> {
     event.preventDefault();
     if (Event.ONCLICK == event.getTypeInt()) {
       AnchorElement el = (AnchorElement) event.getCurrentTarget();
       String href = el.getHref();
-      String placeToken = href.substring(href.indexOf('!'));
-      AppPlaceHistoryMapper appPlaceHistoryMapper =
-        globalAppState.getAppPlaceHistoryMapper();
-      Place newPlace = appPlaceHistoryMapper.getPlace(placeToken);
-      globalAppState.getPlaceChanger().goTo(newPlace);
+      String placeToken = null;
+      // Detect old #!, and direct to the right place
```

Comment:
Here's another general fix.  For markdown that have used relative links that include '#!', the URL that it points to is wrong (so if you hover over it and right click and copy the URL, it will be invalid).  However, if you click it, this code will run, which fixes the link on the fly.